libs/design/modal/README.md

```diff
@@ -1,41 +1,61 @@
-# Modal Component
+# Modal
+Modal is a dynamically rendered element that floats above the rest of a page's content. It can be especially useful for interstitials that require additional user feedback.
 
-The `DaffModalComponent` is a dynamically rendered component that floats above the rest of a page's content. It can be especially useful for interstitials that require additional user feedback.
+## Basic Modal
+<design-land-example-viewer-container example="basic-modal"></design-land-example-viewer-container>
 
-## Usage
+## Supported Content Types
+A modal includes minimally pre-styled components and directives to help structure the content in your modal.
 
-`DaffModalComponent` relies on using the `entryComponents` of the particular Angular `@NgModule`. To use the features of the `DaffModalComponent`:
+### Header
+Header can be added to a modal by using `<daff-modal-header>`. The header includes a title and an optional close button.
 
-* Import the `DaffModalModule` as you would with any other `@daffodil/design` component
-* Add the component that you want rendered inside the modal to your `@NgModule`'s `entryComponents`
+#### Title
+Title can be added to the header by using the `[daffModalTitle]` directive.
+
+#### Close Button
+The close button in the header is shown by default but can be hidden by setting the `dismissible` property to `false` on the header.
+
+### Content
+Content can be added to a modal by using `<daff-modal-content>`. It should only be used once. It's a wrapper container that can be used to place all additional components and text content within a modal. The content container allows for a ton of control and customization because it's minimally pre-styled and serves as a wrapping and spacing container that is scrollable.
+
+### Actions
+Buttons can be added to a modal by using `<daff-modal-actions>`. This container will always be positioned at the bottom of a modal. The horizontal alignment of the actions is set to `end`.
+
+## Configurations
```

Comment:
Remove this section its not longer revelant.

Reply:
removed